content/platforms/kubernetes/kubernetes-architecture.md

```diff
@@ -1,44 +1,55 @@
 
 ![kubernetes-overview-performance-improvements-read]( /images/rs/kubernetes-overview-performance-improvements-read.png )![kubernetes-overview-performance-improvements-write]( /images/rs/kubernetes-overview-performance-improvements-write.png )
 
 ## Multiple services on each Pod
 
-Each Pod includes multiple Redis Enterprise instances (multiple services). We found that the traditional method of deploying a Redis Enterprise database over Kubernetes, in which each Pod includes only a single Redis Enterprise instance while preserving a dedicated CPU, is notably inefficient. Redis Enterprise is exceptionally fast and in many cases can use just a fraction of the CPU resources to deliver the requested throughput. Furthermore, when running a Redis Enterprise Cluster with multiple Redis Enterprise instances across multiple Pods, the Kubernetes network, with its multiple vSwitches, can quickly become the deployment’s bottleneck. Therefore, Redis Labs took a different approach to managing Redis Enterprise over the Kubernetes environment. Deploying multiple Redis Enterprise database instances on a single Pod allows us to better utilize the hardware resources used by the Pod such as CPU, memory, and network while keeping the same level of isolation. See the figure below:
+Redis Labs took a different approach to managing Redis Enterprise over the Kubernetes environment. Each Pod includes multiple Redis Enterprise instances (multiple services). By deploying doing so, this allows us to better utilize the hardware resources used by the Pod such as CPU, memory, and network while keeping the same level of isolation. This is shown on the left side of the figure below.
+
+In our experience, we found that the traditional method of deploying a Redis Enterprise database in Kubernetes as separate pods, preserving a dedicated amount of computed resources (CPU & memory), that contain a single Redis Enterprise instance was notably inefficient. Redis Enterprise is exceptionally fast and in many cases can use just a fraction of the CPU resources to deliver the requested throughput. Furthermore, when running a Redis Enterprise Cluster with multiple Redis Enterprise instances across multiple Pods, the Kubernetes network can quickly become the deployment’s bottleneck. This is show on the right side of the figure below.
```

Comment:
@alexmilowski I'm not clear on why you switched these around. It reads much better when we end this section on the positive note of what RE can do rather than the negative.

Reply:
I think we need to introduce what we did and the benefits without dwelling on the "wrong" approaches. I think I can merge these two.

Reply:
@alexmilowski - agree we should avoid calling is "wrong". Need to compare and contrast the "OSS redis server as a Pod" approach with "Redis Enterprise Cluster node as a Pod" approach